### ISSUE TITLE
Auto teleport in migration 

### ISSUE THREAD

**Issue by exezbcz** [kodadot/nft-gallery#8563]
some calculations are needed there, but imo it will improve the whole experience - it will be in the review step 

two more steps in the signing (teleport and checking funds) 

![image](https://github.com/kodadot/nft-gallery/assets/90852205/3d49f79c-4106-4bef-9e06-a0333cd2112c)

**Comment by exezbcz** [comment]
cc @hassnian 👀

**Comment by exezbcz** [comment]
👀

**Comment by preschian** [comment]
> two more steps in the signing (teleport and checking funds)

@exezbcz, do we need `autoteleport` in the signing step also? Put one in the review step enough, I guess, to avoid redundancy. wdyt?

**Comment by exezbcz** [comment]
@preschian yeah, basically same as with buying with autoteleport - teleport and checking funds for transparency

**Comment by exezbcz** [comment]
there are issues with auto teleport on kusama (ahk -> ksm) so it will be restricted anyway. Putting this issue p4, there are more important issues.
